app/src/org/odk/collect/android/utilities/NepaliDateHelper.java

```diff
@@ -0,0 +1,24 @@
+package org.odk.collect.android.utilities;
+
+import java.util.Date;
+
+import org.commcare.dalvik.R;
+
+import android.content.Context;
+
+/**
+ * Helper to convert a Gregorian Date to a Nepali date, formatted as a 'd MMMM yyyy' string.
+ * 
+ * @author Richard Lu
+ */
+public class NepaliDateHelper {
```

Comment:
This class and the NepaliDateUtilities class both seem fairly similar to me. is there a reason that they are separate?